src/cluster_legacy.c

```diff
@@ -6532,3 +6851,55 @@ int clusterAllowFailoverCmd(client *c) {
 void clusterPromoteSelfToMaster(void) {
     replicationUnsetMaster();
 }
+
+/* Replicate all migrating and importing slot states to a
+ * a given replica client. */
+void clusterReplicateOpenSlots(void)
+{
+    if (!server.cluster_enabled) return;
+
+    int argc = 5;
+    robj **argv = zmalloc(sizeof(robj*)*argc);
+
+    sds cmd = sdsnew("CLUSTER");
+    sds subcmd = sdsnew("SETSLOT");
+    sds importing = sdsnew("IMPORTING");
+    sds migrating = sdsnew("MIGRATING");
+    robj *cmd_obj = createObject(OBJ_STRING, cmd);
+    robj *subcmd_obj = createObject(OBJ_STRING, subcmd);
+    robj *imp_obj = createObject(OBJ_STRING, importing);
+    robj *mig_obj = createObject(OBJ_STRING, migrating);
```

Comment:
The sds objects are used only once, right?

```suggestion
    robj *cmd_obj = createObject(OBJ_STRING, sdsnew("CLUSTER"));
    robj *subcmd_obj = createObject(OBJ_STRING, sdsnew("SETSLOT"));
    robj *imp_obj = createObject(OBJ_STRING, sdsnew("IMPORTING"));
    robj *mig_obj = createObject(OBJ_STRING, sdsnew("MIGRATING"));
```

If we drop the sds variables, we can drop the `_obj` suffix on the objects (if you want).